src/actions.rs

```diff
@@ -43,7 +43,9 @@ pub struct IssueDecorator {
     pub repo_name: String,
     pub labels: String,
     pub assignees: String,
-    pub updated_at: String,
+    pub updated_at: DateTime<Utc>,
```

Comment:
do we need to store both? can we remove the old `updated_at` field?

Reply:
good point. After a doublecheck, it seems that it can be removed :)
(force pushed the change)